README.rst

```diff
@@ -167,32 +167,46 @@ Async image processing
 
  tasks.py
 
+
     .. code :: python
 
-        from django.db.models.loading import get_model
+        try:
+            from django.apps import apps
+            get_model = apps.get_model
+        except ImportError:
+            from django.db.models.loading import get_model
+
+        from celery import shared_task
+
         from stdimage.utils import render_variations
 
-        @app.task()
-        def process_image(app_label, model_name, field_name, file_name):
-            render_variations(app_label, model_name, field_name, file_name)
-            model_class = get_model(app_label, models_name)
-            obj = model_class.objects.get(**{field_name: file_name})
+
+        @shared_task
+        def process_photo_image(file_name, variations, storage):
```

Comment:
Changed Method Name as ist really Can only work with that particular Method

DB Index is redundant, You Arne Right

> Am 02.07.2015 um 13:02 schrieb Johannes Hoppe <notifications@github.com>:
> 
> In README.rst:
> 
> >          from stdimage.utils import render_variations
> >  
> > -        @app.task()
> > -        def process_image(app_label, model_name, field_name, file_name):
> > -            render_variations(app_label, model_name, field_name, file_name)
> > -            model_class = get_model(app_label, models_name)
> > -            obj = model_class.objects.get(**{field_name: file_name})
> > +
> > +        @shared_task
> > +        def process_photo_image(file_name, variations, storage):
> please keep the method name
> 
> —
> Reply to this email directly or view it on GitHub.
>